pandas/tseries/tests/test_timeseries.py

```diff
@@ -941,12 +941,34 @@ def test_nat_vector_field_access(self):
     def test_nat_scalar_field_access(self):
         fields = ['year', 'quarter', 'month', 'day', 'hour',
                   'minute', 'second', 'microsecond', 'nanosecond',
-                  'week', 'dayofyear', 'days_in_month']
+                  'week', 'dayofyear', 'days_in_month', 'daysinmonth',
+                  'dayofweek']
         for field in fields:
             result = getattr(NaT, field)
             self.assertTrue(np.isnan(result))
 
-        self.assertTrue(np.isnan(NaT.weekday()))
+    def test_NaT_methods(self):
+        # GH 9513
+        raise_methods = ['astimezone', 'combine', 'ctime', 'dst', 'fromordinal',
+                         'fromtimestamp', 'isocalendar', 'isoformat',
+                         'strftime', 'strptime',
+                         'time', 'timestamp', 'timetuple', 'timetz',
+                         'toordinal', 'tzname', 'utcfromtimestamp',
+                         'utcnow', 'utcoffset', 'utctimetuple']
+        nat_methods = ['date', 'now', 'replace', 'to_datetime', 'today']
+        nan_methods = ['weekday', 'isoweekday']
+
+        for method in raise_methods:
+            if hasattr(NaT, method):
```

Comment:
Ah, I see your note below now. In that case, I'd slightly prefer to use `if PY3` to adjust the list of tested methods but this is OK too.